app/views/notification_mailer/digest.html.erb

```diff
@@ -1,25 +1,98 @@
-<h2 class='digest-header'>
-  <% if @type == :daily %>
-    Your Notifications From The Past 24h
-  <% elsif @type == :instant %>
-    Your Recent Notifications
-  <% end %>
-</h2>
+<tr>
+  <td>
+    <table cellspacing="0" cellpadding="0">
+      <tr>
+        <td width="640px" align="center" colspan="12">
+          <h2 style="font-weight: 300; margin: 0; width: 100%;">
+            <% if @type == :daily %>
+              Your Notifications From The Past 24h
+            <% elsif @type == :instant %>
+              Your Recent Notifications
+            <% end %>
+          </h2>
+        </td>
+      </tr>
+    </table>
+  </tr>
+</td>
 
-<% @notifications.each do |project, project_notifications| %>
-  <div class='project-wrapper'>
-    <div class='project-name-wrapper'>
-      <span class='project-name'><%= project.name %></span>
-      <hr>
-    </div>
+<tr>
+  <td>
+    <table cellspacing="0" cellpadding="0">
+      <% @notifications.each do |project, project_notifications| %>
+        <tr>
+          <td colspan="12">
+            <table cellspacing="0" cellpadding="0" align="center">
+              <tr>
+                <td height="20px"></td>
+              </tr>
+              <tr>
+                <% if defined?(Dradis::Pro) %>
```

Comment:
We decided we're fine with the Dradis CE project title here. It make the Pro / CE sync easier to manage.

Reply:
✅